condensation-forum/templates/body.html

```diff
@@ -66,14 +66,15 @@
                             <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">
                                 <img src="{{ user.picture }}" style="height: 32px; border-radius: 16px; margin-bottom: -8px; margin-top: -8px;" /> {{user.name}} <span class="caret"></span></a>
                             <ul class="dropdown-menu">
-                                <li><a href="/logout">Sign Out</a></li>
+                                <li class="dropdown-header">{{ user.email }}</li>
```

Comment:
@collinsc This is the entry to add the email. Can you verify your template pulled correctly?

Reply:
@BitFracture I think  I forgot a fetch somewhere in there, I was on an outdated version